README.md

```diff
@@ -3,54 +3,64 @@
 # Static type checker for Python
 
 ### Speed
-Pyright is a fast type checker meant for large Python source bases. It can run in a “watch” mode and performs fast incremental updates when files are modified.

```

Comment:
Please don't change the ".md" files. The changes that you're in this case is a big step backward. Let's restrict prettier to only ".ts" files, not ".md" files.